reference_data/tests_only/simple_gap_model.json

```
@@ -5236,4 +5332,4 @@
   },
   "module_name": "rascal.models.krr",
   "version": "0.1"
-}
+}
```

```suggestion
}

```
(git doesn't like it when files end without newlines)

---

added